docs/docs/get-started/setup-lightdash/install-lightdash.mdx

```diff
@@ -0,0 +1,173 @@
+---|---|---|---|---
+**PGHOST** | string | *auto generated* | false | postgres hostname
+**PGPORT** | number | 5432 | false | Postgres port
+**PGUSER** | string | postgres | false | Postgres user
+**PGPASSWORD** | string |  | true | Postgres password
+**PGDATABASE** | string | postgres | false | Postgres database name
+**SECURE_COOKIES** | boolean | false | false | Secure cookies
+**TRUST_PROXY** | boolean | false | false | Trust the reverse proxy when setting secure cookies (via the "X-Forwarded-Proto" header)
+**LIGHTDASH_SECRET** | string | | true | This is the secret used to sign the session ID cookie and to encrypt sensitive information. **Do not share this secret!**
+**PORT** | number | 8080 | false | Port for Lightdash
+**DBT_PROJECT_DIR** | string | . | false | Path to your local dbt project. Only set this value if you want to use a dbt project from the same machine
+
+
+#### 1. Clone the Lightdash repository
+
+Clone the Lightdash code to your local machine. This will create a new directory called `./lightdash` (the Lightdash directory).
+
+```bash
+# Clone the Lightdash repo
+git clone https://github.com/lightdash/lightdash
+cd lightdash
+```
+
+#### 2. Create containers
+
+```bash
+LIGHTDASH_SECRET="not very secret" PGPASSWORD="password" docker-compose -f docker-compose.yml up --detach --remove-orphans
+```
+
+### Using docker images
+
+:::info
+
+If you're using **podman** replace `docker` with `podman` in the commands mentioned.
+
+:::
+
+#### 1. Start a Postgres container
+
+```bash
+# Example data:
+# Username: postgres
+# Password: mysecretpassword
+# Database name: postgres
+# Port: 5432
+
+docker run --name lightdash-db -p "5432:5432" \
+    -e POSTGRES_PASSWORD=mysecretpassword -d postgres
+```
+
+#### 2. Start Lightdash container
+
+Lightdash environment variables:
+
+Name | Type | Default | Required | Description
+---|---|---|---|---
+**PGHOST** | string |  | true | Postgres hostname
+**PGPORT** | number |  | true | Postgres port
+**PGUSER** | string |  | true | Postgres user
+**PGPASSWORD** | string |  | true | Postgres password
+**PGDATABASE** | string |  | true | Postgres database name
+**LIGHTDASH_SECRET** | string | | true | This is the secret used to sign the session ID cookie and to encrypt sensitive information. **Do not share this secret!**
+**SECURE_COOKIES** | boolean | false | false | Secure cookies
+**TRUST_PROXY** | boolean | false | false | Trust the reverse proxy when setting secure cookies (via the "X-Forwarded-Proto" header)
+:::info
+
+If you want to use a dbt project from the same machine, you will need to set a volume to **/usr/app/dbt**. Example:
+`-v ~/my/dbt/path:/usr/app/dbt`
+
+:::
+
+```bash
+# Example data:
+# DB Host: 192.168.0.1
+# DB Port: 5432
+# DB Username: postgres
+# DB Password: mysecretpassword
+# DB name: postgres
+# Lightdash secret: not very secret
+# Lightdash port: 8080
+
+docker run --name lightdash-app \
+    -e PGHOST=192.168.0.1 -e PGPORT=5432 \
```

Comment:
I wonder what most setups would have here for an IP address? It all depends on the setup :/ one negative of not using docker compose